sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py

```diff
@@ -217,3 +218,29 @@ def is_valid_resource_name(rname: str, exception_type: Optional[Type[BaseExcepti
     if exception_type:
         raise exception_type()
     return False
+
+
+def get_arm_info(cloud_setting: AzureClouds) -> Dict[str, Any]:
```

Comment:
lets rename this to `get_arm_endpoints` - it adds clarity as to what youll get back,